src/components/RichTextEditor/RichTextEditor.stories.tsx

```diff
@@ -108,3 +117,19 @@ export const RichTextEditorFlex: Story<RichTextEditorProps> = (args: RichTextEdi
     </div>
 );
 RichTextEditorFlex.argTypes = { value: { type: "string" } };
+
+export const WithCustomTextStyle: Story<RichTextEditorProps> = (args: RichTextEditorProps) => (
+    <RichTextEditorComponent {...args} />
+);
+WithCustomTextStyle.args = {
+    readonly: false,
+    value: JSON.stringify([
+        ...value,
+        {
+            type: BlockStyleTypes.Paragraph,
+            children: [createLinkNode("https://git-scm.com/downloads", "Link")],
+        },
+    ]),
+    textStyles: [{ type: BlockStyleTypes.H3, className: "tw-text-xxl" }],
```

Comment:
To make the story a bit more explicit, I'd only display some headings and add custom styles for each in order to have a good working example for how to pass in textStyles, e.g.:
```suggestion
    value: JSON.stringify([
        { type: BlockStyleTypes.H1, children: [{ text: headingTitle[BlockStyleTypes.H1] }] },
        { type: BlockStyleTypes.H2, children: [{ text: headingTitle[BlockStyleTypes.H2] }] },
        { type: BlockStyleTypes.H3, children: [{ text: headingTitle[BlockStyleTypes.H3] }] },
        { type: BlockStyleTypes.H4, children: [{ text: headingTitle[BlockStyleTypes.H4] }] },
        { type: BlockStyleTypes.Custom01, children: [{ text: headingTitle[BlockStyleTypes.Custom01] }] },
        { type: BlockStyleTypes.Custom02, children: [{ text: headingTitle[BlockStyleTypes.Custom02] }] },
    ]),
    textStyles: [
        { type: BlockStyleTypes.H1, className: "tw-text-7xl tw-font-bold tw-text-green-80" },
        { type: BlockStyleTypes.H2, className: "tw-text-5xl tw-font-bold tw-text-violet-60" },
        { type: BlockStyleTypes.H3, className: "tw-text-3xl tw-font-bold tw-text-yellow-70" },
        { type: BlockStyleTypes.H4, className: "tw-text-xl tw-text-red-50" },
        { type: BlockStyleTypes.Custom01, className: "tw-font-mono tw-italic tw-text-black-80" },
        { type: BlockStyleTypes.Custom02, className: "tw-underline tw-text-black-80" },
    ],
```

Reply:
nice!